ui/app/routes/vault/cluster/settings/auth/configure.js

```diff
@@ -15,4 +17,12 @@ export default Route.extend({
       return this.store.peekRecord('auth-method', method);
     });
   },
+
+  setupController(controller) {
```

Comment:
removes the need for the global `mountable-auth-methods` helper that was only used in one place